aiida/cmdline/params/types/identifier.py

```diff
@@ -39,45 +48,49 @@ def __init__(self, sub_classes=None):
 
         To prevent having to load the database environment at import time, the actual loading of the entry points
         is deferred until the call to `convert` is made. This is to keep the command line autocompletion light
-        and responsive. The entry point strings will be validated, however, to see if the correspond to known
-        entry points.
+        and responsive. We also postpone the validation of entry point strings for the same reason.
```

Comment:
Let's keep the same tense and mood
```suggestion
        and responsive. The validation of entry point strings is also postponed for the same reason.
```